src/components/InstantSearch.js

```diff
@@ -87,16 +88,29 @@ export default createInstantSearchComponent({
       }),
     };
   },
-  render(createElement) {
-    return createElement(
-      'div',
-      {
-        class: {
-          [this.suit()]: true,
-          [this.suit('', 'ssr')]: false,
+  render(createElementVue2) {
+    if (createElementVue3) {
```

Comment:
could we not do `createElement = createEleementv3 || createElementv2`? This can make the output significantly smaller :)